js/apps/admin-ui/src/clients/advanced/AdvancedSettings.tsx

```diff
@@ -160,6 +165,37 @@ export const AdvancedSettings = ({
               )}
             />
           </FormGroup>
+          {isDPoPEnabled && (
+            <FormGroup
+              label={t("oAuthDPoP")}
+              fieldId="oAuthDPoP"
+              hasNoPaddingTop
+              labelIcon={
+                <HelpItem
+                  helpText={t("clients-help:oAuthDPoP")}
+                  fieldLabelId="clients:oAuthDPoP"
+                />
+              }
+            >
+              <Controller
+                name={convertAttributeNameToForm<FormFields>(
+                  "attributes.dpop.bound.access.tokens"
+                )}
+                defaultValue={false}
+                control={control}
+                render={({ field }) => (
+                  <Switch
+                    id="oAuthDPoP-switch"
+                    label={t("common:on")}
+                    labelOff={t("common:off")}
+                    isChecked={field.value === "true"}
+                    onChange={(value) => field.onChange("" + value)}
+                    aria-label={t("oAuthDPoP")}
```

Comment:
If the `id` here matches the `fieldId`, I believe we do not need to add an `aria-label` attribute.

Reply:
Thank you. I will fix it as you suggested.

Reply:
@tnorimat have you taken a look at this?